.zombienet-tests/utils.js

```diff
@@ -0,0 +1,20 @@
+const { ApiPromise } = require("@polkadot/api")
+
+async function connect(nodeName, networkInfo, isParachain) {
```

Comment:
```suggestion
async function connect(nodeName, networkInfo, parachainId?: number) {
```

Seems better to pass the `100` here, especially since this is a utility file